packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -54,13 +55,66 @@ const FAKE_CID_2 = CID.parse('bafybeig6xv5nwphfmvcnektpnojts44jqcuam7bmye2pb54ad
 const FAKE_CID_3 = CID.parse('bafybeig6xv5nwphfmvcnektpnojts55jqcuam7bmye2pb54adnrtccjlsu')
 const FAKE_CID_4 = CID.parse('bafybeig6xv5nwphfmvcnektpnojts66jqcuam7bmye2pb54adnrtccjlsu')
 const DID_ID = 'did:3:k2t6wyfsu4pg0t2n4j8ms3s33xsgqjhtto04mvq8w5a2v5xo48idyz38l7ydki'
+
 const FAKE_STREAM_ID = StreamID.fromString(
   'kjzl6cwe1jw147dvq16zluojmraqvwdmbh61dx9e0c59i344lcrsgqfohexp60s'
 )
 
-const CONTENT0 = { myData: 0 }
-const CONTENT1 = { myData: 1 }
-const CONTENT2 = { myData: 2 }
+const FAKE_STREAM_ID_EMPTY_SCHEMA = StreamID.fromString(
+  'kjzl6hvfrbw6cbe494mf4cbhmkxbeu5soy2t5iyrs761flvrk5emxpzcjeq5sz4'
+)
+
+const CONTENT_VALID_1 = {
```

Comment:
the changes to this test feel like they are overly complicating this test file and obfuscating the core of what is under test here.  I don't think you need to re-create all the tests that you added in `schema-utils.test.ts`, that is redundant. I think just a simple test to check that some schema validation is run at all would suffice, and then leave the schema-utils test to test the mechanics of schema validation itself.

I'd recommended just having a very simple schema that allows a single `myData` integer field (basically the same format as the original data used in the test before your change), adding a single test for a document that doesn't conform to that simple schema, and otherwise leaving this test file basically as it was